### ISSUE TITLE
TR: PublishedServiceName is missing on some services

### ISSUE THREAD

**Issue by r3to** [openTdataCH/ojp-sdk#46]
`PublishedServiceName` is required on `Service` but missing in some responses

- Example Request: Lausanne - St.Gallen [Gist](https://gist.github.com/r3to/cb4e19434d24364ddbe760bf81d56d39)
- [Schema](https://vdvde.github.io/OJP/develop/index.html#ServiceGroup)

**Comment by TO-mdv** [comment]
Fehler nur bei "Sitzenbleiber". Ticket bereits erhalten "TR Ausgabe Sitzenbleiber korrigieren
[#47](https://github.com/openTdataCH/ojp-sdk/issues/47)

**Comment by r3to** [comment]
```xml
<TimedLeg>
<LegBoard>
  <siri:StopPointRef>ch:1:sloid:3000:501:33</siri:StopPointRef>
  <StopPointName>
    <Text xml:lang="de">Zürich HB</Text>
  </StopPointName>
  <NameSuffix>
    <Text xml:lang="de">NO_DATA</Text>
  </NameSuffix>
  <ServiceDeparture>
    <TimetabledTime>2024-06-10T09:58:00Z</TimetabledTime>
  </ServiceDeparture>
  <Order>1</Order>
</LegBoard>
<LegAlight>
  <siri:StopPointRef>ch:1:sloid:3000:501:34</siri:StopPointRef>
  <StopPointName>
    <Text xml:lang="de">Zürich HB</Text>
  </StopPointName>
  <NameSuffix>
    <Text xml:lang="de">NO_DATA</Text>
  </NameSuffix>
  <ServiceArrival>
    <TimetabledTime>2024-06-10T10:03:00Z</TimetabledTime>
  </ServiceArrival>
  <Order>2</Order>
</LegAlight>
<Service>
  <Mode>
    <PtMode>undefinedModeOfTransport</PtMode>
    <Name>
      <Text xml:lang="de">gesicherter Anschluss</Text>
    </Name>
  </Mode>
  <ConventionalModeOfOperation>scheduled</ConventionalModeOfOperation>
  <TrainNumber/>
  <siri:LineRef>::_x0020_</siri:LineRef>
  <siri:OperatorRef/>
  <ProductCategory>
    <Name>
      <Text xml:lang="de">gesicherter Anschluss</Text>
    </Name>
    <ProductCategoryRef>-1</ProductCategoryRef>
  </ProductCategory>
  <siri:DirectionRef>_x0020_</siri:DirectionRef>
  <OperatingDayRef>2024-06-10</OperatingDayRef>
  <JourneyRef/>
</Service>
</TimedLeg>
```

**Comment by TO-mdv** [comment]
Released on PROD on 27.6.2024

**Comment by r3to** [comment]
✅
